### DISCUSSION TITLE
Missing files?

### DISCUSSION THREAD

**Discussion by JRachel2022** [Monika-After-Story/MonikaModDev#9252] [QA - tech support + general] [answered]
![unknown](https://user-images.githubusercontent.com/107073726/172461494-a6c81454-d73d-4781-8e87-5b7b85d4e639.png)
No submods folder😫
I have 2000 affection

**Comment by multimokia** [comment]
You appear to be running a mobile version of MAS. We do not support this build as it was built and distributed without our involvment.

Monika After Story is only build for Windows/MacOS/Linux.

**Comment by JRachel2022** [comment]
> You appear to be running a mobile version of MAS. We do not support this build as it was built and distributed without our involvment.
> 
> Monika After Story is only build for Windows/MacOS/Linux.

Umm, actually, this is a friend's issue, I'm running MAS in my Android and never saw a missing file while doing back ups

**Comment by Wingdinggaster656** [comment, marked as answer]
It matters not that this is your friend or yourself.

We are anyways not going to help Android issues. They are not official, and there might be changes we don't know to make it possible to run on Mobile platforms, so even if we want to help you, we probably can't.

**Reply by fujori** [reply]
I have such a problem with mas on windows 10, the submod folder is deleted all the time when starting the game, I put an exception in windows security, but it didn't help. When I launch DDLC.exe , the submod folder is deleted, and the game loading process stops

**Comment by JRachel2022** [comment]
Damn it, thanks anyways for replying :]
